File: C/Ex19.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    float v, v0, a, t;
    printf("Digite  a aceleracao em m/s2:");    
    scanf("%f",&a); 

    printf("Digite a velocidade inicial em m/s:");    
    scanf("%f",&v0);

    printf("Digite o tempo do percurso em s:");    
    scanf("%f",&t);

    v=v0+(a*t);
    printf("velocidade final: %.2f m/s \n",v);    

    if(v>60)
    {
      if(v>80)
      {
       if(v>120)
       {
         printf("Veiculo muito rapido");
       }
       else
       {
         printf("Veiculo rapido");  
       }
      }
      else
      {
          printf("Veiculo de cruzeiro");
      } 
    }
    else
    {
        if(v>40)
        {
           printf("Velocidade permitida");
        }
        else
        {
            printf("Veiculo muito lento");
        }
        
    }
    return 0;
}

File: Phyton/Ex19.py
A = float(input("Digite o A: "))
v0= (input("Digite o v0,:"))
t = float(input("Digite o tempo: "))


vf = v0+(A*t)
if(vf<=60):
 if vf <= 40:
     print("Veiculo muito lento")
 else:
     print("Velocidade permitida")

if(vf<=120)and(vf>60):
 if vf <= 80:
     print("Velocidade de cruzeiro")
 else:
     print("Velocidade rápida")
     

else:
  print("Veiculo muito rápida")
  #40 60 80 120